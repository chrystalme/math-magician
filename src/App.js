import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <h2>Math magician Calculator</h2>
        <p>Calculator grid</p>
      </header>
    </div>
  );
}

export default App;
