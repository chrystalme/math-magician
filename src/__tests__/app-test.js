import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../components/app';
import Routes from '../components/routes';

describe('App', () => {
  test('App snapshot test', () => {
    const component = render(
      <Routes>
        <App />
      </Routes>,
    );
    expect(component).toMatchSnapshot();
  });

  test('renders button AC', () => {
    render(
      <Routes>
        <App />
      </Routes>,
    );
    const buttonElement = screen.getByText(/AC/i);
    // eslint-disable-next-line no-unused-expressions
    expect(buttonElement).toBeInTheDocument;
  });
  test('renders button x', () => {
    render(
      <Routes>
        <App />
      </Routes>,
    );
    const buttonElement = screen.getByText(/x/i);
    // eslint-disable-next-line no-unused-expressions
    expect(buttonElement).toBeInTheDocument;
  });
  test('renders button -', () => {
    render(
      <Routes>
        <App />
      </Routes>,
    );
    const buttonElement = screen.getByText(/-/i);
    // eslint-disable-next-line no-unused-expressions
    expect(buttonElement).toBeInTheDocument;
  });
});
