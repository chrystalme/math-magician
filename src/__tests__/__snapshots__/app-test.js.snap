// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar navbar-expand-lg navbar-light bg-light"
      >
        <div
          class="container-fluid"
        >
          <a
            class="navbar-brand"
            href="/"
          >
            Math-magician
          </a>
          <div>
            <ul
              class="navbar-nav"
            >
              <li
                class="nav-item"
              >
                <a
                  aria-current="page"
                  class="nav-link active"
                  href="/"
                >
                  Home
                </a>
              </li>
              <li
                class="nav-item"
              >
                <a
                  class="nav-link"
                  href="/calculator"
                >
                  Calculator
                </a>
              </li>
              <li
                class="nav-item"
              >
                <a
                  class="nav-link"
                  href="/quotes"
                >
                  Quotes
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
      <div
        style="margin: 0px auto;"
      >
        <p
          style="padding: 0px 50px;"
        >
          Lorem ipsum dolor sit amet consectetur, adipisicing elit. At ullam saepe quisquam similique nobis, eligendi, voluptates cupiditate dicta laboriosam, sint sed nam iure excepturi dolores aspernatur! Quas voluptate ea rem molestias, impedit magnam assumenda sapiente delectus ullam aut. Animi harum tenetur quasi qui id sapiente totam laborum natus. Expedita amet, numquam aliquid neque magnam nostrum, maiores, placeat blanditiis corporis error velit. Numquam necessitatibus iste aspernatur, dolorum quod quam saepe sunt sapiente sit porro recusandae veritatis deserunt officia beatae natus ad? Ut iure minima dolor enim, similique hic quas odio. Officia, saepe omnis? Nesciunt maiores nisi, error voluptate ipsum sed aperiam? Dolorum qui reiciendis asperiores, laborum aut nesciunt sequi soluta aliquam unde quidem repellat sunt fuga inventore at hic exercitationem error minima corrupti cum. Autem quibusdam doloribus itaque eius obcaecati numquam quasi dicta culpa nobis amet et saepe ex impedit similique dolore illum accusamus quae pariatur accusantium, voluptatum nostrum quia veniam enim possimus! Hic, cumque. Nesciunt alias esse nulla earum mollitia vel quis, ab dolorem dolor necessitatibus optio dicta. Eveniet delectus mollitia debitis rerum deleniti fugiat, laborum laudantium rem, voluptas temporibus nihil asperiores commodi, molestias ad sapiente eum ut tempora necessitatibus laboriosam iste quidem beatae repellat doloribus? Vitae unde molestias, ipsum quae, facilis rerum repudiandae, iure sequi sed autem commodi suscipit sunt earum minima obcaecati quibusdam voluptas debitis molestiae impedit id excepturi. Aspernatur modi similique eligendi, adipisci voluptatum fuga sunt vel a cum pariatur unde numquam consectetur quam quae ea itaque facilis enim ducimus. Odio minus est consequatur perferendis accusamus obcaecati, natus fuga pariatur reiciendis, laborum, sint odit ea voluptatum nulla voluptatem hic libero? At velit, cum, beatae ipsa distinctio, enim quisquam officiis nisi culpa sit exercitationem. Adipisci nobis officia a minus totam soluta at voluptatem facilis tempora beatae dignissimos, nisi explicabo non in quas ad doloremque earum! Accusamus, numquam dolorem.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar navbar-expand-lg navbar-light bg-light"
    >
      <div
        class="container-fluid"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          Math-magician
        </a>
        <div>
          <ul
            class="navbar-nav"
          >
            <li
              class="nav-item"
            >
              <a
                aria-current="page"
                class="nav-link active"
                href="/"
              >
                Home
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/calculator"
              >
                Calculator
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/quotes"
              >
                Quotes
              </a>
            </li>
          </ul>
        </div>
      </div>
    </nav>
    <div
      style="margin: 0px auto;"
    >
      <p
        style="padding: 0px 50px;"
      >
        Lorem ipsum dolor sit amet consectetur, adipisicing elit. At ullam saepe quisquam similique nobis, eligendi, voluptates cupiditate dicta laboriosam, sint sed nam iure excepturi dolores aspernatur! Quas voluptate ea rem molestias, impedit magnam assumenda sapiente delectus ullam aut. Animi harum tenetur quasi qui id sapiente totam laborum natus. Expedita amet, numquam aliquid neque magnam nostrum, maiores, placeat blanditiis corporis error velit. Numquam necessitatibus iste aspernatur, dolorum quod quam saepe sunt sapiente sit porro recusandae veritatis deserunt officia beatae natus ad? Ut iure minima dolor enim, similique hic quas odio. Officia, saepe omnis? Nesciunt maiores nisi, error voluptate ipsum sed aperiam? Dolorum qui reiciendis asperiores, laborum aut nesciunt sequi soluta aliquam unde quidem repellat sunt fuga inventore at hic exercitationem error minima corrupti cum. Autem quibusdam doloribus itaque eius obcaecati numquam quasi dicta culpa nobis amet et saepe ex impedit similique dolore illum accusamus quae pariatur accusantium, voluptatum nostrum quia veniam enim possimus! Hic, cumque. Nesciunt alias esse nulla earum mollitia vel quis, ab dolorem dolor necessitatibus optio dicta. Eveniet delectus mollitia debitis rerum deleniti fugiat, laborum laudantium rem, voluptas temporibus nihil asperiores commodi, molestias ad sapiente eum ut tempora necessitatibus laboriosam iste quidem beatae repellat doloribus? Vitae unde molestias, ipsum quae, facilis rerum repudiandae, iure sequi sed autem commodi suscipit sunt earum minima obcaecati quibusdam voluptas debitis molestiae impedit id excepturi. Aspernatur modi similique eligendi, adipisci voluptatum fuga sunt vel a cum pariatur unde numquam consectetur quam quae ea itaque facilis enim ducimus. Odio minus est consequatur perferendis accusamus obcaecati, natus fuga pariatur reiciendis, laborum, sint odit ea voluptatum nulla voluptatem hic libero? At velit, cum, beatae ipsa distinctio, enim quisquam officiis nisi culpa sit exercitationem. Adipisci nobis officia a minus totam soluta at voluptatem facilis tempora beatae dignissimos, nisi explicabo non in quas ad doloremque earum! Accusamus, numquam dolorem.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
